include/zephyr/mgmt/mcumgr/mgmt/callbacks.h

```diff
@@ -156,6 +156,8 @@ enum fs_mgmt_group_events {
 	/** Callback when a file has been accessed, data is fs_mgmt_file_access(). */
 	MGMT_EVT_OP_FS_MGMT_FILE_ACCESS		= MGMT_DEF_EVT_OP_ID(MGMT_EVT_GRP_FS, 0),
 
+	MGMT_EVT_OP_FS_MGMT_UPLOAD_DOWNLOAD_DONE	= MGMT_DEF_EVT_OP_ID(MGMT_EVT_GRP_FS, 1),
```

Comment:
`MGMT_EVT_OP_FS_MGMT_FILE_ACCESS_COMPLETE` or
`MGMT_EVT_OP_FS_MGMT_FILE_ACCESS_FINISHED` or `MGMT_EVT_OP_FS_MGMT_FILE_ACCESS_DONE` perhaps so it fits in the alignment?